common/src/main/resources/lang/fr.json

```diff
@@ -86,6 +88,7 @@
   "shape_star": "étoile",
   "shape_triangle": "triangle",
   "task_trim": "%s tronçons supprimés de %s en %s secondes.",
+  "task_trim_update": "Task running for %s. Processed: %s regions (%s%%)",
```

Comment:
Tâche en cours d'exécution pour %s. Traitée : %s régions (%s%%)

Reply:
Added manually. Thank you!